src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/DisasterRecoveryService.cs

```diff
@@ -64,27 +61,49 @@ internal static ConnectionService ConnectionServiceInstance
         /// </summary>
         public void InitializeService(ServiceHost serviceHost)
         {
+            // Get database info
+            serviceHost.SetRequestHandler(ExtendedDatabaseInfoRequest.Type, HandleExtendedDatabaseInfoRequest);
```

Comment:
looks at the rest of the code I'd recommend doing something like the following...

1. Add a method in admin services to get the database info object for a particular db
2. Change this request to be something like "get back config" which calls the method defined in #1
3. Look-up backup specific info in this class
4. return a backup config class that contains a database info object plus the backup config properties.

---
In reply to: [118369150](https://github.com/Microsoft/sqltoolsservice/pull/358#discussion_r118369150) [](ancestors = 118369150)